src/SharedAssemblyInfo.cs

```diff
@@ -4,5 +4,5 @@
 [assembly: AssemblyCompanyAttribute("Akka.NET Team")]
 [assembly: AssemblyCopyrightAttribute("Copyright © 2013-2016 Akka.NET Team")]
 [assembly: AssemblyTrademarkAttribute("")]
-[assembly: AssemblyVersionAttribute("1.0.3.0")]
-[assembly: AssemblyFileVersionAttribute("1.0.3.0")]
+[assembly: AssemblyVersionAttribute("1.0.4.0")]

```

Comment:
To bump the assembly / NuGet verison, you need to edit this file: https://github.com/akkadotnet/Akka.Quartz.Actor/blob/dev/RELEASE_NOTES.md